Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.yml

```diff
@@ -1045,7 +1045,8 @@ script:
     - contextPath: Domain.Name
       description: The domain name.
       type: String
-  dockerimage: demisto/python3:3.7.2.214
+  dockerimage45: demisto/python3:3.7.2.214
```

Comment:
why do we need this one?
did you add something special for 3.8? if not there is no reason for this

Reply:
The validation fails on it.